server/handler/pull_request.go

```diff
@@ -66,6 +67,21 @@ func (h *PullRequest) Handle(ctx context.Context, eventType, deliveryID string,
 		logger.Error().Err(errors.WithStack(err)).Msg("Error processing pull request")
 	}
 
+	if event.GetAction() == "labeled" || event.GetAction() == "opened" {
+		logger.Debug().Msgf("received pull_request %s event", event.GetAction())
```

Comment:
This logging is probably more appropriate higher up in the function where it will apply for all possible actions, rather than only `labeled` and `opened` actions.

Reply:
Great idea, will move this further up